lib/commands/signout.js

```diff
@@ -0,0 +1,23 @@
+const { SignOut } = require('../messages/flow');
+
+const SignedParams = require('../signed-params');
+
+module.exports = async (req, res) => {
+  await res.locals.slackUser.update({
```

Comment:
Nit: since there are a few `res.locals` here, wdyt about pulling them out as variables first so it's obvious what dependencies this has?